src/pybind/mgr/dashboard/__init__.py

```diff
@@ -5,10 +5,36 @@
 """
 
 import os
+import re
+import string
+from typing import NamedTuple
 
 import cherrypy
 
-DEFAULT_VERSION = '1.0'
+DEFAULT_API_VERSION = '1.0'
```

Comment:
Have to  keep this as string yet. As if I do `APIVersion(1,0)` it breaks for some test while `setup_controllers` as `split()` can't work on this type